pkg/meta/base.go

```diff
@@ -741,11 +741,17 @@ func (m *baseMeta) flushQuotas() {
 
 		if err := m.en.doFlushQuotas(Background, quotas); err != nil {
 			logger.Warnf("Flush quotas: %s", err)
+		} else {
```

Comment:
so it's still possible to get wrong quota result? e.g. load usedSpace after flush, then add the `snap.newSpace` again to the `usedSpace`

Reply:
yes, the chance is lower than before